dynawo/sources/Modeler/Common/DYNConnector.h

```diff
@@ -256,7 +256,7 @@ class ConnectorContainer {
    * @param connectorsByVarNum the association between (global) variable index and connector
    * @param flowConnector true if the connector is a flow connector
    */
-  void mergeConnectors(const boost::shared_ptr<Connector>& connector, const boost::shared_ptr<Connector>& reference,
+  void mergeConnectors(boost::shared_ptr<Connector> connector, const boost::shared_ptr<Connector>& reference,
```

Comment:
The const from reference should be removed as I find it confusing (the object pointed by reference is modified in the method)